mage/precommit/mage.go

```diff
@@ -17,9 +17,6 @@ const (
 )
 
 // Precommit runs all the precommit checks.
```

Comment:
Something like this?

```suggestion
// Precommit runs all the precommit checks. Run `mage help:precommit` for information on available options.
```